[Docs] Gitpod CLI - IDP `--decode` flag (#3984)
gitpod-io/website: src/content/docs/configure/workspaces/oidc.md

eyJhbGciOiJSUzI1NiIsImtpZCI6ImlkLTE2ODQ3NTc4MDY...
```

> The output, when decoded, use `--decode` flag to decode, it translates to a JSON object similar to the one below.

Reviewer: ```suggestion
To decode the output, use the `--decode` flag. Once decoded, the result will resemble the JSON object below.
```
